.github/workflows/ci.yml

```diff
@@ -0,0 +1,59 @@
+name: Odin SMR L1 processing CI
+
+on: [push]
+
+jobs:
+    stacks:
+        runs-on: ubuntu-latest
+        steps:
+        - uses: actions/checkout@v2
+
+        - name: Setup Python
+          uses: actions/setup-python@v2
+          with:
+            python-version: '3.10'
+
+        - name: Install General Dependencies
+          run: |
+            python3.10 -m pip install --upgrade pip
+            pip install tox
+
+        - name: Run unit tests
+          run: |
+            tox
+
+    odincal:
+        runs-on: ubuntu-20.04
```

Comment:
the `setup-python` action (and Github as a whole) finally dropped 2.7 support in June, so I had to work around it a bit.